### PR TITLE
kde-misc/plasma-applet-eventcalendar: new package

### PR DESCRIPTION
**Pull request by fedeliallalinea**
Package-Manager: Portage-2.3.79, Repoman-2.3.16
Signed-off-by: Marco Genasci <fedeliallalinea@gmail.com>
Bug: https://bugs.gentoo.org/703620

### CODE REVIEW

**Comment by a17r** [comment]
What does this package add compared to installing applet-window-eventcalendar via e.g. discover? It seems as if there is no build system to actually compile something.

**Comment by fedeliallalinea** [comment]
> What does this package add compared to installing applet-window-eventcalendar via e.g. discover? It seems as if there is no build system to actually compile something.

I think nothing, but why then create an bin ebuild when you can download it and put into /opt?
I like the idea to install all from portage, but I understand that these are personal preferences.

File: kde-misc/plasma-applet-eventcalendar/plasma-applet-eventcalendar-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+DESCRIPTION="Plasma 5 applet for a calendar+agenda with weather that syncs to Google Calendar"
+HOMEPAGE="https://store.kde.org/p/998901/
+	https://github.com/Zren/plasma-applet-eventcalendar"
+
+LICENSE="GPL-2+"
+SLOT="0"
+IUSE=""
+
+DEPEND="
```
Comment (a17r) [inline kde-misc/plasma-applet-eventcalendar/plasma-applet-eventcalendar-9999.ebuild]: If nothing is being built, do we need a `DEPEND` at all?

File: kde-misc/plasma-applet-eventcalendar/plasma-applet-eventcalendar-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	kde-plasma/plasma-workspace:5
+"
+RDEPEND="${DEPEND}"
+
+DOCS=( Changelog.md ReadMe.md )
+
+src_install() {
+	default
+	insinto /usr/share/plasma/plasmoids/org.kde.plasma.eventcalendar
+	doins package/metadata.desktop
```
Comment (a17r) [inline kde-misc/plasma-applet-eventcalendar/plasma-applet-eventcalendar-9999.ebuild]: Use `desktop.eclass` instead.

Comment (fedeliallalinea) [reply kde-misc/plasma-applet-eventcalendar/plasma-applet-eventcalendar-9999.ebuild]: but domenu or newmenu insert in /usr/share/applications. All metadata.desktop are installed in plugin directory (see for example latte-dock)

Comment (a17r) [reply kde-misc/plasma-applet-eventcalendar/plasma-applet-eventcalendar-9999.ebuild]: ah... I missed that.

**Pull request closed without merging**